service/history/replication/metrics_emitter.go

```diff
@@ -173,7 +173,7 @@ func (m *MetricsEmitterImpl) determineReplicationLatency(remoteClusterName strin
 
 	var replicationLatency time.Duration
 	if len(tasks) > 0 {
-		creationTime := time.Unix(0, tasks[0].CreationTime)
+		creationTime := tasks[0].GetVisibilityTimestamp()
```

Comment:
just double checking, does `GetVisibilityTimestamp()` return CreationTime?

Reply:
This is true.